specification/apimanagement/developer-services-api/tspconfig.yaml

```diff
@@ -0,0 +1,38 @@
+parameters:
```

Comment:
@harunrst , please refer to [the latest sample project](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/contosowidgetmanager/Contoso.WidgetManager/tspconfig.yaml) to correct the schema in order to generate the sdk package successfully.